shared/util/forward-logs.desktop.js

```diff
@@ -137,7 +144,7 @@ const setupSource = __STORYBOOK__
       const {ipcRenderer} = require('electron')
       const util = require('util')
 
-      const types = [('log', 'warn', 'error')]
```

Comment:
THIS was preventing console.log from being forwarded - only 'error' was being set up here